resources/build/build-utils.sh

```diff
@@ -509,10 +548,21 @@ builder_parse() {
     shift # past the processed argument
   done
 
-  # TODO: not sure if this is appropriate or if we should error?
   if (( ! ${#_builder_chosen_action_targets[@]} )); then
     for e in "${_builder_targets[@]}"; do
-      _builder_chosen_action_targets+=("build$e")
+      _builder_chosen_action_targets+=("$_builder_default_action$e")
+    done
+  fi
```

Comment:
From https://github.com/keymanapp/keyman/pull/6986#issuecomment-1196035602:

> Now that I look at this, this is probably not right -- it applies build to all targets, not just the ones (if any) specified on the command line...

Reply:
Yeah but if you look at line 551, you'll see this is only run if no targets have been specified on the command line. Ah bash logic FTW!

Reply:
... if no action+target pairings have been made thus far.  That's different from "no targets", isn't it?

Example:
```
./build.sh :engine
```
This has no actions specified, but there is a target specified.

Granted, the way the script is _currently_ written, this is handled by auto-assigning the loose target an action, but I have concerns with how that is handled at present; addressing those may cause this concern to re-arise here.  (See next comment chain.)

Reply:
"Thus far" -- this is the last statement in the function (apart from debug).